.gitignore

```diff
@@ -47,6 +47,11 @@ local.properties
 #
 node_modules/
 npm-debug.log
+yarn.lock
+
+# iOS/CocoaPods
+#
+ios/Pods
```

Comment:
The ignoring of cocoapods-related directories is outside the subject of this PR. Moreover, cocoapods is no longer in the latest master. Please remove the modification.